python/ray/dashboard/client/src/pages/dashboard/node-info/NodeInfo.tsx

```diff
@@ -12,12 +12,31 @@ import {
 } from "@material-ui/core";
 import React from "react";
 import { connect } from "react-redux";
+import { RayletInfoResponse } from "../../../api";
 import { StoreState } from "../../../store";
 import Errors from "./dialogs/errors/Errors";
 import Logs from "./dialogs/logs/Logs";
 import NodeRowGroup from "./NodeRowGroup";
 import TotalRow from "./TotalRow";
 
+const clusterWorkerPids = (
+  rayletInfo: RayletInfoResponse,
+): Map<string, Set<string>> => {
+  // Given a Raylet response, this extracts, per node, all the worker pids registered
```

Comment:
I found this sort of hard to understand - maybe "Groups PIDs registered with each raylet by node IP address." ?